common/anomalies/hfe_anomaly.txt

```diff
@@ -111,7 +112,7 @@ hfe_anomaly_dead_worms = {
 			years_passed > 90
 		}
 		modifier = {
-			factor = 2
+			factor = 5
```

Comment:
Next have factor 10 but this one is not

Reply:
As the years get farther and farther, the multiplication rates go up. Each one is higher than the last.

Reply:
Never mind.

Okay, done.